DOCS/man/mpv.rst

```diff
@@ -781,9 +781,11 @@ ignored. This Lua code execution is not sandboxed.
 
 Any variables in condition expressions can reference properties. If an
 identifier is not already defined by Lua or mpv, it is interpreted as property.
-For example, ``pause`` would return the current pause status. If the variable
-name contains any ``_`` characters, they are turned into ``-``. For example,
-``playback_time`` would return the property ``playback-time``.
+For example, ``pause`` would return the current pause status. You can not
+reference properties with ``-`` as usual since that would denote a subtraction,
```

Comment:
since this is a special mechanism there is no "as usual"
I suggest "this way"

Reply:
I changed it, but with "as usual" I was referring to input.conf and scripts